examples/resources/awscc_ec2_vpn_gateway/ec2_vpn_gateway.tf

```diff
@@ -0,0 +1,8 @@
+resource "awscc_ec2_vpn_gateway" "example_vpn_gateway" {
+  type = "ipsec.1"
+  tags = [{
+    key   = "Name"
+    value = "Example VPN Gateway"
+  }]
```

Comment:
probably add this 

tags = [{
    key   = "Modified By"
    value = "AWSCC"
  }]